stable/goldilocks/values.yaml

```diff
@@ -17,7 +17,7 @@ image:
   # image.repository -- Repository for the goldilocks image
   repository: us-docker.pkg.dev/fairwinds-ops/oss/goldilocks
   # image.tag -- The goldilocks image tag to use
-  tag: v4.5.0
+  tag: v4.5.1
```

Comment:
do you want to switch to using `appVersion`? And maybe unpin patch?

Reply:
at some point, maybe